### PR TITLE
Edit KYC err msg

### PR DESCRIPTION
**Pull request by zaelgohary**
Related issue: https://github.com/threefoldtech/threefold_connect/issues/771

### CODE REVIEW

File: app/lib/screens/identity_verification_screen.dart [hunk 1]
```diff
@@ -1276,7 +1276,7 @@ class _IdentityVerificationScreenState
                 image: Icons.warning,
                 title: 'Not enough balance',
                 description:
-                    'Please fund your account with at least $minimumBalance TFTs.',
+                    'Please fund your Daily TFChain wallet with at least $minimumBalance TFTs.',
```
Comment (AhmedHanafy725) [inline app/lib/screens/identity_verification_screen.dart]: user may change the wallet name

Comment (zaelgohary) [reply app/lib/screens/identity_verification_screen.dart]: Done.

File: app/lib/screens/identity_verification_screen.dart [hunk 2]
```diff
@@ -1265,6 +1266,11 @@ class _IdentityVerificationScreenState
                 ],
               ));
     } on NotEnoughBalance catch (_) {
+      final wallet = (await getPkidWallets())
+          .where((w) => w.type == WalletType.NATIVE)
+          .toList()
+          .first
```
Comment (AhmedHanafy725) [inline app/lib/screens/identity_verification_screen.dart]: what if the wallet is not initialized and the list is empty?

Comment (zaelgohary) [reply app/lib/screens/identity_verification_screen.dart]: Handled.

File: app/lib/screens/identity_verification_screen.dart [hunk 3]
```diff
@@ -1265,6 +1266,8 @@ class _IdentityVerificationScreenState
                 ],
               ));
     } on NotEnoughBalance catch (_) {
+      final wallets = await getPkidWallets();
+      final wallet = wallets.firstWhere((w) => w.type == WalletType.NATIVE);
```
Comment (AhmedHanafy725) [inline app/lib/screens/identity_verification_screen.dart]: `firstWhere` will crash if the list is empty, please use `where`

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: app/lib/screens/identity_verification_screen.dart (hunk 3 above)
Comment (zaelgohary) [reply app/lib/screens/identity_verification_screen.dart]: fixed.

File: app/lib/screens/identity_verification_screen.dart [hunk 4]
```diff
@@ -1250,8 +1253,9 @@ class _IdentityVerificationScreenState
                 type: DialogType.Warning,
                 image: Icons.warning,
                 title: 'Not enough balance',
-                description:
-                    'Please fund your account with at least $minimumBalance TFTs.',
+                description: wallets.isEmpty
```
Comment (AhmedHanafy725) [inline app/lib/screens/identity_verification_screen.dart:1257]:
```suggestion
                description: wallet.isEmpty
```

Comment (zaelgohary) [reply app/lib/screens/identity_verification_screen.dart:1257]: combined wallets & wallet in one variable.

File: app/lib/screens/identity_verification_screen.dart [hunk 5]
```diff
@@ -1250,8 +1254,9 @@ class _IdentityVerificationScreenState
...
+                    ? 'Please, initialize a wallet and fund it with at least $minimumBalance TFTs.'
+                    : 'Please, fund your ${wallets.first.name} TFChain wallet with at least $minimumBalance TFTs.',
```
Comment (AhmedHanafy725) [inline app/lib/screens/identity_verification_screen.dart]:
```suggestion
                    ? 'Please initialize a wallet and fund it with at least $minimumBalance TFTs.'
                    : 'Please fund your ${wallets.first.name} TFChain wallet with at least $minimumBalance TFTs.',
```

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**